input-method.php

```diff
@@ -14,7 +15,8 @@
 ?>
 <Response>
     <Gather numDigits="1" timeout="10" action="input-method-result.php?SearchType=<?php echo $searchType ?>" method="GET">
+        <Say voice="<?php echo $voice; ?>" language="<?php echo $language; ?>"><?php echo $inputTitle ?></Say>
```

Comment:
Since this is option, you should wrap this in an `if` block.  

Per my previous comment pull the title from the respective configuration value.